### ISSUE TITLE
Pasting multiline input with escapes doesn't work.

### ISSUE THREAD

**Issue by gothrek22** [sedwards2009/extraterm#164]
Hi,

On windows + wsl pasting line end escaped content doesn't actually line escape, treating the paste as simple multiline.

F.e. pasting this:
`yum install -y yum-utils \
  device-mapper-persistent-data \
  lvm2`

splits it into 3 commands, instead of one multiline input.

**Comment by sedwards2009** [comment]
What are you pasting from? What does the original look like?

**Comment by gothrek22** [comment]
Sometimes from the browser, sometimes from vscode. I see that github didn't keep the formatting. The command originally looked like this:

```
yum install -y yum-utils \
 device-mapper-persistent-data \
 lvm2
```

**Comment by sedwards2009** [comment]
Fixed in 0.41.1.
